### PR TITLE
91 feature rejected place chekurkov

### PR DESCRIPTION
**Pull request by Dmitry528**
1) Open server, client, admin - on this branch
2) Add new place from client side ( on profile page )
3) Reject this place on admin panel
4) And in admin panel you can see this place in Rejected-places section

### CODE REVIEW

File: src/components/approve-table/table-layout.vue [hunk 1]
```diff
@@ -54,6 +63,21 @@ export default {
             if(event.target.value === 'Approve'){
                 event.target.value = 'Approved'
             }
+        },
+        onSubmith: async function(event, rowId){
+            if(this.reason === ''){
```
Comment (marrybbb) [inline src/components/approve-table/table-layout.vue]: suggestion:
if(!this.reason)

Comment (Dmitry528) [reply src/components/approve-table/table-layout.vue]: Done

**Review by Viktor-Hordynskyi — APPROVED** [review]
Tested, works well

**Review by andrii-kondratiuk — APPROVED** [review]
It works. Approved.

File: src/services/user.service.js [hunk 2]
```diff
@@ -62,6 +62,41 @@ const UserService = {
...
+            const token = TokenService.getToken();
+            const headers = {
+                Authorization: token,
+                'Content-Type': 'application/json'
+            }
+            const response = await RequestService.put('/api/reject', body, headers);
+        }
+        catch(error){
+            throw new Error(error);
+        }
```
Comment (marrybbb) [inline src/services/user.service.js:99]: Suggestion:
All methods for places are written in user-service file in which all methods related to users written. It is better to create separate file for those methods)

Comment (Dmitry528) [reply src/services/user.service.js:99]: Done)

**Review by marrybbb — APPROVED** [review]

**Pull request merged**